test/gtest/common/test_obj_size.cc

```diff
@@ -57,7 +57,7 @@ UCS_TEST_F(test_obj_size, size) {
     EXPECTED_SIZE(uct_base_ep_t, 8);
     EXPECTED_SIZE(uct_rkey_bundle_t, 24);
     EXPECTED_SIZE(uct_self_ep_t, 8);
-    EXPECTED_SIZE(uct_tcp_ep_t, 160);
+    EXPECTED_SIZE(uct_tcp_ep_t, 272);
```

Comment:
I guess we should consider replacing `struct sockaddr_storage` by pointer to that type as future memory consumption optimization (in the separate PR). It will behave as follows:
1. Allocate `peer_addr` in EP from the heap.
2. When connection established (i.e. EP is moved to CONNECTED state), decide what to do with it:
- if DEBUG build. don't free the `peer_addr` and it will be released by EP dtor
- otherwise - `free()` and set to `NULL`. all logging function must be updated to print `<no debug data>` string if `peer_addr == NULL`
@bureddy @yosefe @brminich wdyt?